53.md

```diff
@@ -0,0 +1,49 @@
+NIP-53
+======
+
+Calendar Event RSVPs
+--------------------
+
+`draft` `optional` `author:tyiu`
+
+This specification defines RSVPs representing a response to a calendar event per [NIP-52](52.md) to indicate a user's attendance intention.
+
+If a calendar event tags a pubkey, that can be interpreted as the calendar event creator inviting that user to attend. Clients MAY choose to prompt the user to RSVP for the calendar event.
+
+Any user may RSVP, even if they were not tagged on the calendar event. Clients MAY choose to prompt the calendar event creator to invite the user who RSVP'd. Clients also MAY choose to ignore these RSVPs.
+
+This NIP is intentionally not defining who is authorized to attend a calendar event if the user who RSVP'd has not been tagged. It is up to the calendar event creator to determine the semantics.
+
+This NIP is also intentionally not defining what happens if a calendar event changes after an RSVP is submitted.
+
+### Format
+
+The format uses a parameterized replaceable event kind `31925`.
+
+The `.content` of these events is optional and should be a free-form note that adds more context to this calendar event response.
+
+The list of tags are as follows:
+* `a` (required) reference tag to kind `31923` calendar event being responded to per [NIP-33](33.md)
+* `d` (required) universally unique identifier. Generated by the client creating the calendar event RSVP.
+* `L` (required) label namespace of `nip54.status` per [NIP-32](32.md)
```

Comment:
As of writing, NIP-32 has not yet been merged to master, so at the moment, this links to nothing. Here's the PR:
https://github.com/nostr-protocol/nips/pull/532